content/docs/admin/mail-config.md

```diff
@@ -0,0 +1,92 @@
++++
+title = "Mail Configuration"
+description = "Configuring Bookstack to send mail"
+date = "2017-08-08"
+type = "admin-docs"
+slug = "mail-config"
++++
+
+This document has been written by referencing the file at `config/mail.php`.
```

Comment:
I'd prefer not to mention the `config/mail.php` file to prevent config changes that will be overwritten upon update/